src/index.js

```diff
@@ -37,6 +48,26 @@ function addTimelineMeasures(pluginInstance) {
   return true;
 }
 
+function metricsFromModuleAutoData(plugin) {
```

Comment:
A more descriptive name for this function? What is the `module` in this context?

Reply:
Better description is good, changing